premium.php

```diff
@@ -967,6 +1024,25 @@ function premium() {
 		</form>
 		</td></tr>";
 
+		$genders = array('Male', 'Female','Non-binary','None');
```

Comment:
Let's make this array a static var on the User class. For reference, see: https://github.com/levimeahan/shinobi-chronicles/blob/ce15896764593a63a48892b7f393d3c7a0408fd6/classes/User.php#L188-L196

But you can do the constants instead of plain strings in defining that array

Reply:
done